pkg/core/native/management.go

```diff
@@ -615,7 +615,7 @@ func checkScriptAndMethods(script []byte, methods []manifest.Method) error {
 	offsets := bitfield.New(l)
 	for i := range methods {
 		if methods[i].Offset >= l {
-			return errors.New("out of bounds method offset")
+			continue
```

Comment:
C# node bug, please.

Reply:
And btw, I think it'll be compatible with `methods[i].Offset > l`.

Reply:
> C# node bug, please.

https://github.com/neo-project/neo/issues/2767. But technically, such script can be safely handled by both C# and Go VMs, see https://github.com/nspcc-dev/neo-go/blob/80823c58d96a8a2542df1c1e92b57f7af7f67e24/pkg/vm/context.go#L108.

>  it'll be compatible with `methods[i].Offset > l`

Why? Only for the current case, but for other possible cases it won't, see https://github.com/neo-project/neo-vm/blob/a65487fa56be3eccb2c1dbfec5dcdd71b8a05fde/src/Neo.VM/Script.cs#L146.

Reply:
I keep underestimating C# node.